src/server/config.rs

```diff
@@ -125,6 +130,7 @@ impl Default for Config {
             labels: HashMap::default(),
             advertise_addr: DEFAULT_ADVERTISE_LISTENING_ADDR.to_owned(),
             status_addr: DEFAULT_STATUS_ADDR.to_owned(),
+            advertise_status_addr: DEFAULT_ADVERTISE_LISTENING_ADDR.to_owned(),
```

Comment:
advertise_addr and advertise_status_addr has the same default value, will them cause errors?

Reply:
It won't, the default value is an empty string, and both `advertise_addr` and `advertise_status_addr` will fall back to the listening address if they are empty.